pgrx-pg-sys/build.rs

```diff
@@ -760,7 +763,7 @@ fn env_tracked(s: &str) -> Option<String> {
 }
 
 fn target_env_tracked(s: &str) -> Option<String> {
-    let target = std::env::var("TARGET").unwrap();
+    let target = env_tracked("TARGET").unwrap();
```

Comment:
I'm surprised that we need to add `env_tracked` to this, I thought Cargo was already tracking this? Are we sure we won't confuse Cargo on this point?